src/kernel-install/kernel-install.in

```diff
@@ -189,19 +189,16 @@ for suff in $ENTRY_TOKEN_SEARCH; do
         else
             [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "$pref/$suff not found…"
         fi
-    done
-done
-
-[ -z "$BOOT_ROOT" ] && for pref in "/efi" "/boot" "/boot/efi"; do
-    if [ -d "$pref/loader/entries" ]; then
-        BOOT_ROOT="$pref"
 
-        [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && \
-            echo "$pref/loader/entries exists, using BOOT_ROOT=$BOOT_ROOT"
-        break
-    else
-        [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "$pref/loader/entries not found…"
-    fi
+        if [ -d "$pref/loader/entries" ]; then
+            BOOT_ROOT="$pref"
```

Comment:
Don't we need to conditionalize this on `[ -z BOOT_ROOT ]` as well?

Reply:
Force pushed with this added.

Effectively it doesn't actually matter: if we get to this point and BOOT_ROOT is set, $pref must be the same, so we'd just reset the variable to the same value. But I think it's more elegant to have the check.